challenges/2021_cloud_base_height/04_eval_pytorch.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>what are the options for refering to this in a nicer way, maybe registering a model?</p><p>Also it would be good to ciopy your artifacts / database somewhere central so tohers can also access it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I've copied my artifacts into the /projects/ folder under "ml_artifacts" in the cbh challenge directory.

I feel that registering a model will help, but there still has to be a manual selection of the model you want to evaluate at some point in the notebook, and retrieving the registered model also turned out to be less straightforward than desired. So think this solution is passable atm. I added some explainer text on how to reproduce this manual selection, though